images/linux/scripts/installers/aws-iam-authenticator.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+################################################################################
+##  File:  aws-iam-authenticator.sh
+##  Desc:  Installs AWS IAM Authenticator
+################################################################################
+
+source $HELPER_SCRIPTS/install.sh
+
+# Determine latest aws-iam-authenticator CLI version
+AWS_IAM_AUTHENTICATOR_LATEST_VERSION_URL=https://api.github.com/repos/kubernetes-sigs/aws-iam-authenticator/releases/latest
```

Comment:
I suggest installing this plugin in `installers/aws.sh` instead of creating new script

Reply:
that works for me, I will add this into aws.sh then.